packages/appkit/src/universal-adapter/client.ts

```diff
@@ -423,6 +424,10 @@ export class UniversalAdapterClient {
       JSON.stringify(this.appKit?.getCaipNetwork())
     )
 
+    if (this.appKit?.getCaipAddress() && !this.appKit?.getIsConnectedState()) {
```

Comment:
We have `getIsConnected` and `getIsConnectedState`, they seem to do the same things, we should remove one of them, and the other should return `ChainController.state.activeCaipAddress` , if there is an address being set we are connected, otherwise we are not. And we stopped using `setIsConnected`